modules/virt-configuring-default-cpu-model.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-configuring-default-cpu-model.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-default-cpu-model_{context}"]
+= Configuring the default CPU model
+
+Configure the `defaultCPUModel` by updating the `HyperConverged` custom resource (CR). You can change the `defaultCPUModel` while OpenShift Virtualization is running.
+
+Note: The `defaultCPUModel` is case sensitive.
```

Comment:
this should be formatted like other asciidoc admonitions across the docs using block syntax:
```suggestion
[NOTE]
====
The `defaultCPUModel` is case sensitive.
====
```